recipes/serpent/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.12" %}
```

Comment:
Seems like there is a 1.12.1 release, but it is not on PyPI. Will that be uploaded soon @irmen?